chroma-manager/tests/framework/integration/upgrade_os/cluster_setup

```diff
@@ -24,6 +24,12 @@ USE_FENCE_XVM=false
 
 eval $(python $CHROMA_DIR/chroma-manager/tests/utils/json_cfg2sh.py "$CLUSTER_CONFIG")
 
+# We need CLIENT_1 to exist after cluster_setup
+if [ -z ${CLIENT_1} ]; then
+    echo "Missing lustre_clients in cluster configuration"
+    exit 1
+fi
```

Comment:
`${CLIENT_1}` needs to be `""` quoted here.  If it is empty, it will be a syntax error.  But per below, I don't think adding this as a solution to proposed requirement [below](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/474/files/14be42727b015aed13b9c647828d20850e656e37..1e351342c757fd8d23e3c859bde4959423de95ff#diff-4cbeadfa244f4bc2f8ecf2934b297766R41) is the right way to go.